### PR TITLE
Main menu

### PR DESCRIPTION
**Pull request by artaksafaryan**
Made menu responsible.

### CODE REVIEW

**Review by mikayel00 — APPROVED** [review]
like

File: css/header.css [hunk 1]
```diff
@@ -73,7 +79,29 @@
...
+}
+
+#nav_icon1{
+    display: none;
+}
+
+#nav_icon1 span{
+    display: block;
+    margin:4px;
+    height: 9px;
```
Comment (robhov97) [inline css/header.css]: avoid odd numbers in spacings and sizing

File: css/header.css [hunk 2]
```diff
@@ -73,7 +79,29 @@
...
+    width:60px;
+    background: #bdbdbd;
+    border-radius: 9px;
```
Comment (robhov97) [inline css/header.css]: same

**Review by robhov97 — CHANGES_REQUESTED** [review]

**Review by lilithkarapetyan — APPROVED** [review]
After resolving the issues Robert mentioned, feel free to merge the PR

**Pull request merged**